test/apicoverage-webhook/Dockerfile

```diff
@@ -0,0 +1,23 @@
+# Copyright 2019 The Knative Authors
```

Comment:
If this docker image is generic, it should live in test-infra, not here. If it's serving-specific, it should not live in knative-tests GCR.

Reply:
Each repo might add files that are specific to that repo. Here we are adding ignored_fields.yaml that is specific per repo, which makes having the image in the repo more sense. Which GCR would you suggest here?

Reply:
Synced offline, building the image at runtime will be addressed in a future PR.